designsafe/static/scripts/data-depot/services/published.service.js

```diff
@@ -160,7 +160,8 @@ export class PublishedService {
         }
 
         // Check for reports
-        if(has(resp.data, 'reports')) entities.push(...resp.data.reports);
+        const isSimulation = has(resp.data, 'simulations') || has(resp.data, 'hybrid_simulations');
```

Comment:
We already inspect `resp.data` for `simulations` and `hybrid_simulations` above. Could you set `isSimulation` there?